query-engine/connector-test-kit-rs/query-engine-tests/tests/new/write_conflict.rs

```diff
@@ -0,0 +1,77 @@
+use indoc::indoc;
+use query_engine_tests::*;
+
+#[test_suite(schema(schema), only(MongoDb))]
+mod write_conflict {
+    use futures::future;
+    use query_engine_tests::Runner;
+
+    fn schema() -> String {
+        let schema = indoc! {
+            r#"model Invoice {
+              #id(id, Int, @id)
+              text String?
+            }
+            "#
+        };
+
+        schema.to_owned()
+    }
+
+    #[connector_test]
+    async fn simple(runner: Runner) -> TestResult<()> {
+        let futs: Vec<_> = (0..50)
```

Comment:
Is this enough concurrency to *always* guarantee that the test would fail without the implemented fix for retrying?

Reply:
I honestly can't say. Has it consistently reproduced locally so far and on CI on this PR? Yes. Is it enough to _always_ guarantee, I can't say. Best I can offer atm is to wait and see, as always for those types of tests.